.gitlab/ci/on-push.yml

```diff
@@ -189,6 +190,7 @@ mpv2-prepare-testing-bucket:
     MARKETPLACE_VERSION: "marketplacev2"
     MARKETPLACE_BUCKET: "$GCS_MARKET_V2_BUCKET"
     PRODUCTNAME: "Cortex XSIAM"
+    ENV_DEMISTO_SDK_MARKETPLACE: "$MARKETPLACE_VERSION"
```

Comment:
Shouldn't we add this key to the: `xpanse-prepare-testing-bucket` as well?

Reply:
not ready yet, this is testing phase.

Reply:
@daryakoval changed the implementation, you are welcome to CR :)